Stripe/STPPaymentContext.m

```diff
@@ -568,7 +568,7 @@ - (void)requestSourceFlowRedirectPayment:(STPSource *)source {
     STPSourceCompletionBlock onRedirectCompletion = ^(STPSource *finishedSource, NSError *error) {
         stpDispatchToMainThreadIfNecessary(^{
             if (error) {
-                [self didFinishWithStatus:STPPaymentStatusError error:error];
+                [self didFinishWithStatus:STPPaymentStatusPending error:nil];
```

Comment:
Are there any categories of error that we want to return as StatusError here? Is it possible to call into this and never actually present the user the dialog in a way we can know about? (eg if there is no network connection).